src/tests/ftest/launch.py

```diff
@@ -345,14 +345,12 @@ def run_command(cmd):
 
     """
     print("Running {}".format(" ".join(cmd)))
-    try:
-        with subprocess.Popen(
-                cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
-                universal_newlines=True) as process:
-            stdout, _ = process.communicate()
-            retcode = process.poll()
-    except Exception as error:
-        raise RuntimeError("Error executing '{}':\n\t{}".format(" ".join(cmd), error))
+
+    process = subprocess.Popen(
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations